### PR TITLE
Document GitHub serverside limit on URL length

### LINKED ISSUE #5136: Document GitHub serverside limit on URL length
### What article on docs.github.com is affected?

https://docs.github.com/en/github/managing-your-work-on-github/about-automation-for-issues-and-pull-requests-with-query-parameters

### What part(s) of the article would you like to see updated?

Document the serverside limit of 8191 bytes URL.

### PR DESCRIPTION
**Pull request by gukoff**
[example of a long URL](https://github.com/azure/azure-iot-cli-extension/issues/new?template=Bug_report.md&body=%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A%0A%23%23%23+%2A%2AThis+is+autogenerated.%0A123123213)

### Why:

**Closes #5136**

### What's being changed:

Add a note about the URL length limit.

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by amilcaru123 — APPROVED** [review]

**Comment by janiceilene** [comment]
Thanks for opening a PR 👍 Please fill out the [pull request template](https://raw.githubusercontent.com/github/docs/main/.github/PULL_REQUEST_TEMPLATE.md) in the top comment in this pull request, then we'll get this up for review!

**Review by Abhishek577270 — APPROVED** [review]
I got gold money cash

**Comment by guaxy94** [comment]
Yes You see friend, i'm really happy yo help u

**Review by Abhishek577270 — APPROVED** [review]

**Comment by gukoff** [comment]
> Thanks for opening a PR 👍 Please fill out the pull request template in the top comment in this pull request, then we'll get this up for review!

OK, will do

**Review by Abhishek577270 — DISMISSED** [review]

**Review by Abhishek577270 — COMMENTED** [review]
My love

**Comment by gukoff** [comment]
Duplicated info from here to https://github.com/github/docs/issues/5136
Added here a link to this new issue.
Don't see what else to change in the template, it's a very small change.

**Comment by janiceilene** [comment]
Thanks for filling out the PR template and opening an issue @gukoff! I'll get this up for review ⚡  Can you clarify how you know the serverside limit is 8191 bytes URL?

**Comment by gukoff** [comment]
> Can you clarify how you know the serverside limit is 8191 bytes URL?

Sure, see the link "example of a long URL" in the description.

File: content/github/managing-your-work-on-github/about-automation-for-issues-and-pull-requests-with-query-parameters.md [hunk 1]
```diff
@@ -19,7 +19,11 @@ Query parameters are optional parts of a URL you can customize to share a specif
 
 You must have the proper permissions for any action to use the equivalent query parameter. For example, you must have permission to add a label to an issue to use the `labels` query parameter.
 
-If you create an invalid URL using query parameters, or if you don’t have the proper permissions, the URL will return a 404 error page.  
+### Errors
+
+If you create an invalid URL using query parameters, or if you don’t have the proper permissions, the URL will return a `404 Not Found` error page.  
+
+If you create an URL longer than 8191 bytes, the URL will return a `414 URI Too Long` error page.
```
Comment (lee-dohm) [inline content/github/managing-your-work-on-github/about-automation-for-issues-and-pull-requests-with-query-parameters.md:26]:
```suggestion
If you create an invalid URL using query parameters, or if you don’t have the proper permissions, the URL will return a `404 Not Found` error page. If you create a URL that exceeds the server limit, the URL will return a `414 URI Too Long` error page.
```

Simplifying the language and bringing it into line with our standards.

**Review by lee-dohm — APPROVED** [review]
Looks good!

**Pull request merged**